test/benchmarks/README.md

```diff
@@ -18,8 +16,7 @@ Using this example, one would simply run
 
     jmh:runMain scala.collection.mutable.OpenHashMapRunner
```

Comment:
not sure how to run only a single benchmark, `sbt jmh:run` runs everything, even if I provide a pattern after it :/